### DISCUSSION TITLE
Prisma & Postgres sometimes "Can't reach database server"

### DISCUSSION THREAD

**Discussion by gopeter** [vercel/community#4501] [Help]
### Summary

From time to time I get an connection error when trying to connect to my Vercel Postgres data base with Prisma.

I'm running just the Hobby plan for now and wonder if the database "goes to sleep" sometimes?

### Steps to Reproduce

Error

```
Can't reach database server at `....eu-central-1.postgres.vercel-storage.com`:`5432`","","Please make sure your database server is running at `....eu-central-1.postgres.vercel-storage.com`:`5432`
```

That's my config:

```
datasource db {
  provider = "postgres"
  url = env("DATABASE_PRISMA_URL")
  directUrl = env("DATABASE_URL_NON_POOLING")
}
```

**Comment by amyegan** [comment]
This sounds like it might be related to [cold starts](https://vercel.com/docs/storage/vercel-postgres/limits#vercel-postgres-cold-starts). A Vercel Postgress database can be in either an `Active` or `Idle` state. After 5 minutes with no active queries, it will go to the `Idle` state to save compute resources when it isn't in use. 

There are ways to [prevent Vercel Postgres from entering an `Idle` state](https://vercel.com/docs/storage/vercel-postgres/faq#prevent-vercel-postgres-from-entering-an-idle-state), just remember that it will use additional compute hours. 

https://vercel.com/docs/storage/vercel-postgres/faq#vercel-postgres-connection-time
